<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClustalOSupportRunDialog</class>
 <widget class="QDialog" name="ClustalOSupportRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>269</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Align with Clustal Omega</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Iteration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QCheckBox" name="maxGTIterationsCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Max number guidetree iterations</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="maxGTIterationsSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="iterationNumberCheckBox">
          <property name="text">
           <string>Number of iterations</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="maxHMMIterationsCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Max number of HMM iterations</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="maxHMMIterationsSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="iterationNumberSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of CPUs being used</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberOfCPUSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="setAutoCheckBox">
          <property name="text">
           <string>Set options automatically</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="alignButton">
       <property name="text">
        <string>Align</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>maxGTIterationsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxGTIterationsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterationNumberCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxGTIterationsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterationNumberCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxHMMIterationsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxHMMIterationsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxHMMIterationsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterationNumberCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>iterationNumberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
